core/src/components/cat-tooltip/cat-tooltip.tsx

```diff
@@ -0,0 +1,68 @@
+import {Component, h, Host, Prop} from '@stencil/core';
+import {autoUpdate, computePosition, flip, offset, Placement} from "@floating-ui/dom";
+
+@Component({
+  tag: 'cat-tooltip',
+  styleUrl: 'cat-tooltip.scss',
+  shadow: true,
+})
+export class CatTooltip {
+  private tooltip?: HTMLElement;
+  private trigger?: Element;
+
+  /**
+   * The content of the tooltip
+   */
+  @Prop() content!: string;
```

Comment:
Don't use `!` for properties. We cannot make sure that the property is actually given by the developer. Instead, you need to handle it in code

Reply:
Please make sure that the tooltip is properly updated if the content changes while the tooltip is shown